UnitsNet/QuantityFormatter.cs

```diff
@@ -124,6 +124,12 @@ public static string Format<TUnitType>(IQuantity<TUnitType> quantity, string for
         /// <returns>The string representation.</returns>
         public static string Format<TUnitType>(IQuantity<TUnitType> quantity, string format, IFormatProvider? formatProvider)
             where TUnitType : Enum
+        {
+            return FormatUntrimmed(quantity, format, formatProvider).Trim();
```

Comment:
Not a huge difference, but I think this could be simplified to `TrimEnd`

Reply:
Fixed